tests/conftest.py

```diff
@@ -201,13 +211,35 @@ def account_keys():
 
 
 @pytest.fixture(scope="module")
-def env(gas_limit, evm_version, evm_backend, tracing, account_keys) -> BaseEnv:
+def exporter(request: FixtureRequest):
+    export_dir_str = request.config.getoption("export")
+    if not export_dir_str:
+        yield None
+        return
```

Comment:
can this just be return?
```suggestion
        return
```

Reply:
i don't think it can. pytest treats fixtures with `yield` as generator fixtures and expects them to yield a value